extensions/volcano/generator-v1beta1/cmd/generate/generate.go

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package main
+
+import (
+	"fmt"
+	"github.com/fabric8io/kubernetes-client/generator/pkg/schemagen"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"reflect"
+	volcanov1beta1 "volcano.sh/apis/pkg/apis/scheduling/v1beta1"
+)
+
+func main() {
+
+	// the CRD List types for which the model should be generated
+	// no other types need to be defined as they are auto discovered
+	crdLists := map[reflect.Type]schemagen.CrdScope{
+		reflect.TypeOf(volcanov1beta1.PodGroupList{}):   schemagen.Namespaced,
```

Comment:
gofmt is off here... please run `gofmt -w extensions/volcano/generator-v1beta1/cmd/generate/generate.go`

Reply:
Thanks! Done.